launchable/commands/helper.py

```diff
@@ -20,6 +20,14 @@ def find_or_create_session(context, session: Optional[str], build_name: Optional
     if session:
         return session
     elif build_name:
+        if not read_build():
+            raise click.UsageError(click.style(
+                "Make sure to run `launchable record build --name {}` before, \nOr this command was ran on another machine what was ran `launchable recrod build` command. So please use --session option to run".format(build_name), fg="yellow"))
```

Comment:
TODO: note to myself; @Konboi and I discussed this part and there will be some change here, so review this message afterward.

Reply:
I finished to change, so could you please review again?